test/babel6/stylesheet-registry.js

```diff
@@ -2,7 +2,7 @@
 import test from 'ava'
 
 // Ours
-import StyleSheetRegistry from '../../src/stylesheet-registry'
+import { StyleSheetRegistry } from '../../src/stylesheet-registry'
```

Comment:
Since this is going to be a major release probably we can drop support for Babel 6